src/design_notes/copy_ergonomics.md

```diff
@@ -0,0 +1,105 @@
+# Copy type ergonomics
+
+## Background
+
+There are a number of pain points with `Copy` types that the lang team is
+interested in exploring, though active experimentation is not currently ongoing.
+
+Some key problems are:
+
+## `Copy` cannot be implemented with non-`Copy` members
+
+There are standard library types where the lack of a `Copy` impl is an
+active pain point, e.g., [`MaybeUninit`](https://github.com/rust-lang/rust/issues/62835)
+and [`UnsafeCell`](https://github.com/rust-lang/rust/issues/25053)
```

Comment:
```suggestion
and [`UnsafeCell`](https://github.com/rust-lang/rust/issues/25053) (if the contained type is itself `Copy`).
```

Reply:
I would like to expand this to explain the (possible) relation to unsafe impl Copy.